subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r42/ToolingApiDeprecatedClientJvmCrossVersionSpec.groovy

```diff
@@ -55,10 +56,19 @@ mainClassName = 'TestClient'
 """
         file('src/main/java/TestClient.java') << """
 import org.gradle.tooling.GradleConnector;
+import java.io.File;
 
 public class TestClient {
-    public static void main(String[] args) {
-        GradleConnector.newConnector();
+    public static void main(String[] args) throws Exception {
```

Comment:
Previously, the code `GradleConnector.newConnector()` didn't execute any build at all, thus the report couldn't be generated because the rendering action happened after a build.